networkit/cpp/centrality/TopCloseness.cpp

```diff
@@ -148,8 +148,8 @@ void TopCloseness::computeReachableNodesDir() {
   }
 
   for (count v = 0; v < n; v++) {
-    reachL[v] = reachL_scc[sccs.componentOfNode(v) - 1];
-    reachU[v] = reachU_scc[sccs.componentOfNode(v) - 1];
```

Comment:
It is more a question than direct feedback: Does omitting `-1` has side-effects in other parts of the code. `TopHarmonicCloseness.cpp` still refers to sccs with `-1`. In `CutClustering.cpp` connected components (without strongly) are used without the index-shift. At least there is some inconsistency in how `componentOfNode` is used for different types of components.

Reply:
I checked other parts of the codebase, `TopCloseness` was the only one to shift the component ids for `StronglyConnecteedComponents`. This was done because the lowest component id from `StronglyConnectedComponents` was 1. Now it is 0 (same as `ConnectedComponents`), so the id shift is not necessary anymore.

Reply:
Ok, I understand. However on branch `Dev` there is at least one reference  for  `-1` in `TopHarmonicCloseness` (but maybe this is intended then). 
https://github.com/networkit/networkit/blob/Dev/networkit/cpp/centrality/TopHarmonicCloseness.cpp#L467

Reply:
Yes I addressed that in this PR.

Reply:
Oh boy, somehow I missed that. Thanks for clarification